### PR TITLE
Cleaned up if/else branches

### PR DESCRIPTION
**Pull request by anmousyon**
More readable code still functions the same

### CODE REVIEW

File: reddit_liveupdate/utils.py [hunk 1]
```diff
@@ -10,30 +10,23 @@
...
-            tzinfo=pytz.UTC,
-            format="dd MMM YYYY HH:mm",
-            locale=c.locale,
-        )
+        date_format="dd MMM YYYY HH:mm"
+
+    return format_datetime(
+        datetime=dt,
+        tzinfo=pytz.UTC,
+        date_format="dd MMM YYYY HH:mm",
```
Comment (spladug) [inline reddit_liveupdate/utils.py]: shouldn't this be referencing the new variable?

**Comment by spladug** [comment]
[:nail_care:](https://github.com/reddit/reddit/wiki/The-Code-Salon)

Thanks for your contribution!

File: reddit_liveupdate/utils.py (hunk 1 above)
Comment (anmousyon) [inline reddit_liveupdate/utils.py]: Oh yes, sorry. Had to refork and copy paste some stuff. Must've gotten lost in that. I'll fix that really quick.

**Comment by anmousyon** [comment]
image

**Pull request closed without merging**